src/processors/ProcessorStore.cpp

```diff
@@ -53,6 +53,8 @@
 #if BYOD_ENABLE_ADD_ON_MODULES
 #include <AddOnProcessors.h>
 #include <AddOnProcessorStore.h>
+
+
 #endif
```

Comment:
I did a faulty merge in one of my commits and ended up with some of your changes in them. Clang format keeps changing how these are formatted and makes it think there's a change. Is there an easy way to get rid of this change?